sdk/docs/iot/README.md

```diff
@@ -1,82 +1,61 @@
+From a functional perspective, this means that the user's application code (not the SDK) calls directly to the MQTT stack of their choice. The SDK provides utilities (functions, default values, etc.) that help make the connection and feature set easier to use. Some examples of those utilities include:
 
-- Publish topics to which messages can be sent and subscription topics to which users can subscribe for incoming messages.
-- Functions to parse incoming message topics which populate structs with crucial message information.
+- Publish topics to which messages can be sent.
+- Subscription topics to which users can subscribe for incoming messages.
+- Functions to parse incoming message topics, which populate structs with crucial message information.
 - Default values for MQTT connect keep alive and connection port.
 
-A full list of features can be found in the doxygen docs listed below in [Docs](#docs).
+To better understand the responsibilities of the user application code and the Embedded C SDK, please take a look at the [State Machine diagram](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/docs/iot/mqtt_state_machine.md), which explains the high-level architecture, SDK components, and a clear view of SDK x Application responsibilities.
+
+For more details about the Azure IoT Hub, see the following [Azure IoT Hub documentation](https://docs.microsoft.com/azure/iot-hub/).
+For more details about the Azure IoT Device Provisioning Service (DPS), see the [Azure IOT DPS documentation](https://docs.microsoft.com/azure/iot-dps/).
 
-**Note**: this therefore requires a different programming model as compared to the earlier version of the C SDK ([found here](https://github.com/Azure/azure-iot-sdk-c)). To better understand the responsibilities of the user application code and the SDK, please take a look at the [State Machine diagram](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/docs/iot/mqtt_state_machine.md) that explains the high-level architecture, SDK components, and a clear view of SDK x Application responsibilities.
+**NOTE**: This Embedded C SDK requires a different programming model as compared to the earlier version of the [C SDK](https://github.com/Azure/azure-iot-sdk-c).
 
-### Docs
+### Documentation
 
-For API documentation, please see the doxygen generated docs [here][azure_sdk_for_c_doxygen_docs]. You can find the IoT specific docs by navigating to the **Files -> File List** section near the top and choosing any of the header files prefixed with `az_iot_`.
+For API documentation, please see the [doxygen generated documents](https://azuresdkdocs.blob.core.windows.net/$web/c/docs/1.0.0/index.html). After following that link, you can find the IoT specific documents by navigating to the **Files -> File List** section near the top and choosing any of the header files prefixed with `az_iot_`.
```

Comment:
Nit: "doxygen generated" - for end user I don't think they care about whether it was doxygen or not, but just the documentation.  Could we just say "See [here] for API documentation"?  Similar feedback on other doxygen generated if there are any.